### ISSUE TITLE
Caption missing in Table Component 

### ISSUE THREAD

**Issue by deleonio** [ant-design/ant-design#23675]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/ant-design/ant-design](https://github.com/ant-design/ant-design)

### Steps to reproduce
no option exists

### What is expected?
We need a caption tag for accessibility

http://www.edition-w3.de/TR/1999/REC-html401-19991224/struct/tables.html#h-11.4

### What is actually happening?
no caption tag

| Environment | Info |
|---|---|
| antd | 4.1.5 |
| React | 16 |
| System | Windows |
| Browser | Chrome |

---
thank you

**Comment by martinoppitz** [comment]
https://github.com/ant-design/ant-design/issues/18934

**Comment by mjfwebb** [comment]
English reference for those interested: https://www.w3.org/TR/html401/struct/tables.html#h-11.2.2

**Comment by deleonio** [comment]
Caption is required for accessibility / screenreader

**Comment by plmsz** [comment]
Hi, is this something that could be implement in the near future?
I've tried to render a caption with the property title:
`<Table columns={columns} dataSource={data} title={() => <caption>Teste</caption>} />`

But it's useless, since the NVDA doesn't read as caption.

**Comment by GiuZambot** [comment]
> Hi, is this something that could be implement in the near future? I've tried to render a caption with the property title: `<Table columns={columns} dataSource={data} title={() => <caption>Teste</caption>} />`
> 
> But it's useless, since the NVDA doesn't read as caption.

**Comment by GiuZambot** [comment]
@plmsz I agree, we need to think more about accessibility in general

**Comment by KaraAJC** [comment]
Bumping this back up, as it's really needed
